website/scripts/fetch-remote-filepaths.ts

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable no-console */
+import fs from 'node:fs/promises'
+import path from 'node:path'
+import prettier from 'prettier'
+
+type Params = {
+  user: string
+  repo: string
+  branch: string
+  docsPath: string
+  outputPath: string
+  filterDocs?: (filePath: string) => boolean
+}
+
+async function fetchRemoteFilePaths({ user, repo, branch, docsPath, outputPath, filterDocs }: Params): Promise<void> {
```

Comment:
@benface this fn will avoid GitHub rate limit :)

Reply:
Nice!